grails-app/services/org/pih/warehouse/core/DocumentTemplateService.groovy

```diff
@@ -150,7 +151,8 @@ class DocumentTemplateService {
                 quantity: v?.sum { it.quantity },
                 totalPrice: v?.sum { it.totalPrice() },
                 budgetCode: v?.budgetCode?.first()?.code ?: "",
-                expectedShippingDate: v?.estimatedReadyDate?.first() ? v?.estimatedReadyDate?.first()?.format(Constants.EUROPEAN_DATE_FORMAT) : ""
+                expectedShippingDate: v?.estimatedReadyDate?.first() ? v?.estimatedReadyDate?.first()?.format(Constants.EUROPEAN_DATE_FORMAT) : "",
```

Comment:
have you tried what will happen if estimatedReadyDate is null? I remember that `.first()` in Groovy is broken and even if there is a null safe operator it can throw an error.

Reply:
I have not changed anything here (except adding a comma), so I assume this must have been tested and is working properly.